### ISSUE TITLE
Allow connection by IP also

### ISSUE THREAD

**Issue by jersten** [CiscoCloud/terraform.py#45]
I have noticed that by default I have to connect by DNS name; it would be nice to be able to pass a switch (maybe `--ipv4`) and connect directly by IP instead of using DNS.

**Comment by sean-abbott** [comment]
+1

**Comment by sean-abbott** [comment]
Along with #47, where I suggested config options, I would like to default to private IP without needing any tags.

**Comment by sean-abbott** [comment]
I modified the name assignment of aws_host to achieve this in a hacked fashion: 

`name = resource['primary']['attributes']['private_ip']`

But a more permanent solution would be nice.

**Comment by stevendborrelli** [comment]
I agree with this. It would be nice to have the option of using private IPs.

We probably also need to rethink options so that ENV vars can be used, since ansible will not send arguments down to the node.

**Comment by sean-abbott** [comment]
ENV vars are ok, but would prefer config options from a code-sharing standpoint.  I find things that require you source env vars before running to be more fraught with peril than stuff that allows you give a one liner to your coworkers.
